sdks/python/scripts/generate_pydoc.sh

```diff
@@ -117,6 +118,8 @@ master_doc = 'index'
 html_theme = 'sphinx_rtd_theme'
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 project = 'Apache Beam'
+version = version.__version__
```

Comment:
does this not cause a name collision with the package version? I guess we aren't using (package) version again

Reply:
Sorry, I'm not sure I understand. Could you give an example where this becomes an issue?

Reply:
I don't think it actually causes an issue right now since nothing is getting used later, just makes me nervous when we have variables with the same names as packages

Reply:
Got it. Sure, I'll change the alias 👍